mindmeld/cli.py

```diff
@@ -85,6 +91,112 @@ def _app_cli(ctx):
         ctx.obj = {}
 
 
+def _dvc_add_helper(filepath):
+    """
+    Returns True if successful, False otherwise along with helper message
+    :param path:
+    :return:
+    """
+    p = subprocess.Popen(
+        ["dvc", "add", filepath], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    # Get DVC error message from standard error
+    _, error = p.communicate()
+    error_string = error.decode("utf-8")
+
+    if DVC_INIT_ERROR_MESSAGE in error_string:
+        return False, DVC_INIT_HELP
+    elif DVC_ADD_DOES_NOT_EXIST_MESSAGE in error_string:
+        return False, DVC_ADD_DOES_NOT_EXIST_HELP.format(dvc_add_path=filepath)
```

Comment:
If the error_string contained some other type of error, we would return True here, but shouldnt we also raise this error to the client?

Reply:
Yeah good catch, I'll have it return the error string if it exists.